source/nodejs/ac-typed-schema/src/markdown/generate-markdown.ts

```diff
@@ -164,9 +164,20 @@ function createTable(formattedProperties: any[]) {
 	*/
 	var tableData = [];
 	var headerRow = [];
+
 	for (let propName in formattedProperties[0]) {
 		headerRow.push(__(propName));
 	}
+
+	// Used for custom table rendering.
+	// We append the table title (wrapped in braces) to the first cell in the header,
+	// and the table renderer will use this value for aria-label.
+	// It is then removed from the header.
+	var firstValue = headerRow[0];
+	if (firstValue) {
+		headerRow[0] = `{${title}}` + firstValue;
```

Comment:
Like you, I'm not a big fan of this approach... However, to do this in a more structured way may require a lot of work (that would itself likely be pretty flaky).

That said, we should probably wrap the title more cleanly... 
  - Consider using `{{` and `}}` as your delimiters
  - Perhaps even use `<!-- TableTitle: "Title goes here" -->`?
  - When parsing the title out in the renderer, instead of assuming that the title exists and begins at index `10`, look for the delimiters and pull them out that way -- this helps ensure that this functionality is robust to changes in `marked` output